src/Faker/Provider/et_ET/Address.php

```diff
@@ -0,0 +1,829 @@
+<?php
+
+namespace Faker\Provider\et_ET;
+
+class Address extends \Faker\Provider\Address
+{
+    protected static $cityPrefix = array('linn');
+
+    protected static $regionSuffix = array('maakond');
+    protected static $streetSuffix = array(
+        'tn', 'tee', 'mnt', 'allee','maantee', 'puiestee','põik', 'käik', 'tänav'
+    );
+
+
+    protected static $postcode = array('#####');
+
+    /**
+    * 'Estonia' in different languages
+    */
+    protected static $eesti = array(
+                            'Eesti',
+                            'Estonia',
+                            'Estland',
+                            'Igaunija',
+                            'Viro',
+                            'Estija',
+                            'Эстония');
+    
+    /**
+     * @link https://et.wikipedia.org/wiki/Maailma_riikide_loend
+     */
+    protected static $country = array(
+                            'Afganistan',
```

Comment:
Can you write all countries starting with the same letter in a single line? It will reduce the line count of this file.